js/main.js

```diff
@@ -33,20 +37,42 @@ const handleNavItemsAnimation = () => {
 	})
 }
 
-const targetDate = new Date('2023-09-15T16:00:00.000Z')
+const targetDate = new Date('2023-09-15T16:00:00.000+0100')
```

Comment:
przy zmianie czasu nic się nie stanie?

Reply:
jak na razie tylko tak doszedłem do tego, aby pokazywało że o 16 będzie wesele, więc po zmianie czasu na letni, sprawdzi się jak to wygląda, i ewentualnie da małą korektę :)